tests/CollateralRiskSteward.t.sol

```diff
@@ -0,0 +1,449 @@
+
+  function setUp() public virtual override {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 21974363);
+    riskConfig = DeployHorizonRiskStewards._getRiskConfig();
+    steward = new RiskSteward(
+      address(AaveV3Ethereum.POOL),
+      AaveV3Ethereum.CONFIG_ENGINE,
+      riskCouncil,
+      GovernanceV3Ethereum.EXECUTOR_LVL_1,
+      riskConfig
+    );
+
+    vm.prank(GovernanceV3Ethereum.EXECUTOR_LVL_1);
+    AaveV3Ethereum.ACL_MANAGER.addRiskAdmin(address(steward));
+  }
+
+  /* ----------------------------- Caps Tests ----------------------------- */
+  /// should revert due to config
+  function test_updateCaps() public virtual override {
+    (uint256 daiBorrowCapBefore, uint256 daiSupplyCapBefore) = AaveV3Ethereum
+      .AAVE_PROTOCOL_DATA_PROVIDER
+      .getReserveCaps(AaveV3EthereumAssets.DAI_UNDERLYING);
+
+    IEngine.CapsUpdate[] memory capUpdates = new IEngine.CapsUpdate[](1);
+    capUpdates[0] = IEngine.CapsUpdate(
+      AaveV3EthereumAssets.DAI_UNDERLYING,
+      (daiSupplyCapBefore * 110) / 100, // 10% relative increase
+      (daiBorrowCapBefore * 110) / 100 // 10% relative increase
+    );
+
+    // without adhering to minDelay, DebounceNotRespected
+    vm.startPrank(riskCouncil);
+    vm.expectRevert(IRiskSteward.DebounceNotRespected.selector);
+    steward.updateCaps(capUpdates);
+
+    (uint256 daiBorrowCapAfter, uint256 daiSupplyCapAfter) = AaveV3Ethereum
+      .AAVE_PROTOCOL_DATA_PROVIDER
+      .getReserveCaps(AaveV3EthereumAssets.DAI_UNDERLYING);
+
+    RiskSteward.Debounce memory lastUpdated = steward.getTimelock(
+      AaveV3EthereumAssets.DAI_UNDERLYING
+    );
+    assertEq(daiBorrowCapAfter, daiBorrowCapBefore);
+    assertEq(daiSupplyCapAfter, daiSupplyCapBefore);
+    assertEq(lastUpdated.supplyCapLastUpdated, 0);
+    assertEq(lastUpdated.borrowCapLastUpdated, 0);
+
+    // after min time passed test caps decrease
+    vm.warp(MIN_DELAY);
+    (daiBorrowCapBefore, daiSupplyCapBefore) = AaveV3Ethereum
+      .AAVE_PROTOCOL_DATA_PROVIDER
+      .getReserveCaps(AaveV3EthereumAssets.DAI_UNDERLYING);
+
+    capUpdates[0] = IEngine.CapsUpdate(
+      AaveV3EthereumAssets.DAI_UNDERLYING,
+      (daiSupplyCapBefore * 90) / 100, // 10% relative decrease
+      (daiBorrowCapBefore * 90) / 100 // 10% relative decrease
+    );
+    // after adhering to minDelay, UpdateNotInRange
+    vm.expectRevert(IRiskSteward.UpdateNotInRange.selector);
+    steward.updateCaps(capUpdates);
+    vm.stopPrank();
+
+    (daiBorrowCapAfter, daiSupplyCapAfter) = AaveV3Ethereum
+      .AAVE_PROTOCOL_DATA_PROVIDER
+      .getReserveCaps(AaveV3EthereumAssets.DAI_UNDERLYING);
+
+    assertEq(daiBorrowCapAfter, daiBorrowCapBefore);
+    assertEq(daiSupplyCapAfter, daiSupplyCapBefore);
+  }
+
+  function test_updateCaps_outOfRange() public virtual override {
+    vm.warp(MIN_DELAY);
+    super.test_updateCaps_outOfRange();
+  }
+
+  function test_updateCaps_debounceNotRespected() public virtual override {
```

Comment:
i think this is not needed and the first one can be renamed to smt like `test_updateCaps_reverts`, but all good
or are we overriding all existing tests only to make them work with new config?